src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java

```diff
@@ -207,7 +209,14 @@ private String getFilePathSuffix() {
         public Collection<? extends SCMDescriptor<?>> getApplicableDescriptors() {
             StaplerRequest req = Stapler.getCurrentRequest();
             Job<?,?> job = req != null ? req.findAncestorObject(Job.class) : null;
-            return SCM._for(job);
+            List<SCMDescriptor<?>> scms = new ArrayList<>();
+            for(SCMDescriptor<?> scm : SCM._for(job)) {
+                if(!"org.jenkinsci.plugins.multiplescms.MultiSCM".equals(scm.getId())) {
+                    scms.add(scm);
+                }
+            }
+            return scms;
+
```

Comment:
or more concisely
```suggestion
            return SCM._for(job).stream().filter(d -> !"org.jenkinsci.plugins.multiplescms.MultiSCM".equals(d.getId())).collect(Collectors.toList());
```
or similar, plus fix imports